src/editor.rs

```diff
@@ -137,7 +137,7 @@ impl<'a> Editor<'a> {
     /// # Errors
     ///
     /// Will return `Err` if an error occur when enabling termios raw mode, creating the signal hook
-    /// or when obtaining the terminal window size.
```

Comment:
`new` will return `Err` if an error occurs when obtaining the terminal window size initially (`editor.update_window_size()?`)

Reply:
Oh, this make more sense :-)

Reply:
Oh, I see now.  Thank you.